### PR TITLE
refactor(/caniuse/feature): simplify route

### PR DESCRIPTION
**Pull request by sidvishnoi**

### CODE REVIEW

File: routes/caniuse/caniuse.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+import { createResponseBodyHTML, sanitizeBrowsersList } from "./lib/index.js";
+
+type Params = { feature: string };
+type Query = { browsers?: string };
+type IRequest = Request<Params, any, any, Query>;
+
+export default async function route(req: IRequest, res: Response) {
+  const feature = req.params.feature;
+  const browsers = sanitizeBrowsersList(req.query.browsers?.split(/,/g) || []);
```
Comment (marcoscaceres) [inline routes/caniuse/caniuse.ts:13]: Q: what's the difference between `/,/g` and just `.split(",")`?

**Review by marcoscaceres — APPROVED** [review]
In case you come back to this...

**Comment by marcoscaceres** [comment]
Same with this one, @sidvishnoi ... whatcha wanna do with it?

File: routes/caniuse/caniuse.ts (hunk 1 above)
Comment (sidvishnoi) [reply routes/caniuse/caniuse.ts:13]: Hmm I was probably planning to use `/,\s*/`, but let's stick with `","` to keep it simple (and hope people form the request right).

Comment (sidvishnoi) [inline routes/caniuse/caniuse.ts:13]:
```suggestion
  const browsers = sanitizeBrowsersList(req.query.browsers?.split(",") || []);
```

**Comment by sidvishnoi** [comment]
I'll check if we can and need to support some old JSON-based routes...

**Review by marcoscaceres — APPROVED** [review]
Lgtm

**Comment by sidvishnoi** [comment]
We use JSON response in xref. This was removing it 😬 
This PR is not needed as we already have what we use.

**Pull request closed without merging**